docs/quick-start.mdx

```diff
@@ -0,0 +1,347 @@
+---
+sidebar_label: Quick Start
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
```

Comment:
Hm, I am torn about mdx personally, for two reasons:

* We probably want to keep docs readable in the source form as well
* We'll probably migrate from docusaurus to something with less javascript, and it not clear whethre that (as of yet unknown) new thing would support mdx

Reply:
That's fair.

I think MDX fits this use case nicely because we want to have different flavors of instructions for different OS's. I think the tab structure would also work well for combining all of the client library docs into a single page (@jorangreef had suggested that).